frontend/src/views/camp/Dashboard.vue

```diff
@@ -180,14 +180,21 @@ export default {
       const authUser = this.$auth.user()
       const activityResponsibles = scheduleEntry.activity().activityResponsibles().items
       return activityResponsibles.some(
-        (ar) => ar.campCollaboration().user().id === authUser.id
+        (ar) => this.userIdOrInviteEmail(ar) === authUser.id
       )
     },
-    sortCampCollaborations(campCollaborations) {
-      return campCollaborations.sort(
-        (a, b) => parseInt(a.user().id, 16) - parseInt(b.user().id, 16)
+    sortActivityResponsibles(activityResponsibles) {
+      return activityResponsibles.sort(
```

Comment:
I was stumbling over this recently when trying to debug the reactivity issues with vue2.7:
`Array.sort()` performs the sorting algorithm in place, so we're actually directly mutating items in the vuex store. This is not very nice.

Better to use `lodash.sortBy()` or copy the array before sorting `[...activityResponsibles].sort()`

Reply:
Fixed in 20a8ef9